Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -357,6 +357,9 @@ YUI.add('ez-binarybase-editview', function (Y) {
          */
         _prepareDrop: function (e) {
             e.preventDefault();
+            if (this.get('isNotTranslatable')) {
```

Comment:
I would do this check before preventing the default behavior and not calling `e.preventDefault()` if drag and drop is not handled